.circleci/config.yml

```diff
@@ -63,7 +63,7 @@ commands:
 jobs:
   build-client-integration:
     docker:
-      - image: $AWS_ACCOUNT_ID.dkr.ecr.us-east-1.amazonaws.com/ef-cms-us-east-1:2.9.0
+      - image: $AWS_ACCOUNT_ID.dkr.ecr.us-east-1.amazonaws.com/ef-cms-us-east-1:2.11.0
```

Comment:
Was this the only job whose container version was updated on purpose?

Reply:
Holy cow, nope! Good catch, thank you!

Reply:
Deploying to exp5 again

Reply:
https://app.circleci.com/pipelines/github/flexion/ef-cms/39253/workflows/c6cc905d-1ed4-413e-9212-3dee6178cb7f